data-prepper-plugins/drop-events-processor/README.md

```diff
@@ -1,6 +1,15 @@
 # Drop Events Processor
 This is a processor that drops all messages that are passed into it.
 
+## Changes
+
+### Starting in Data Prepper v1.3.0
```

Comment:
The `drop` processor is brand-new in 1.3.0, so we don't need to discuss changes.